### PR TITLE
App Popup

### PR DESCRIPTION
**Pull request by OlexiyDobroskok**
### HTML і CSS практика: Hooli-style Popupe

[Demo](https://olexiydobroskok.github.io/app__Popup/) |
[Code base](https://github.com/OlexiyDobroskok/app__Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/OlexiyDobroskok/app-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+              <label for="popup__icon" class="chek__label">
+                <span class="popup__checkbox"></span>
+              </label>
+              <div class="popup__container">
+                <ul class="popup__list">
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img
+                        src="./image/user_icon.png"
+                        alt="user"
```
Comment (A-Ostrovnyy) [inline submissions/OlexiyDobroskok/app-Popup/index.html:29]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/OlexiyDobroskok/app-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="menu">
+          <ul class="menu__list">
+            <li class="menu__item"><a href="#" class="menu__link">About</a></li>
+            <li class="menu__item"><a href="#" class="menu__link">Foto</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/OlexiyDobroskok/app-Popup/index.html:17]:
```suggestion
            <li class="menu__item"><a href="#" class="menu__link">Photo</a></li>
```

File: submissions/OlexiyDobroskok/app-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                  <input
+                    type="checkbox"
+                    id="popup"
+                    class="popup__checkbox"
+                    tabindex="0"
+                  />
+                  <label for="popup" class="popup__button">
+                    <span class="popup__button__more">MORE</span>
+                    <span class="popup__button__hide">HIDE</span>
+                  </label>
```
Comment (A-Ostrovnyy) [inline submissions/OlexiyDobroskok/app-Popup/index.html:115]: Check your markup in [validator](https://validator.w3.org/)

File: submissions/OlexiyDobroskok/app-Popup/index.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                        alt="search"
+                        class="popup__img"
+                      />Search
+                    </a>
+                  </li>
+                  <input
+                    type="checkbox"
+                    id="popup"
+                    class="popup__checkbox"
+                    tabindex="0"
```
Comment (A-Ostrovnyy) [inline submissions/OlexiyDobroskok/app-Popup/index.html:110]: This line makes no sense because input already has focus by default.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome!

**Comment by A-Ostrovnyy** [comment]
@OlexiyDobroskok Why did you close PR?

**Comment by OlexiyDobroskok** [comment]
i have some problem with git, i can open again?

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by OlexiyDobroskok** [comment]
> 

Thnx =)

**Pull request merged**